app/serializers/project_serializer.rb

```diff
@@ -0,0 +1,7 @@
+class ProjectSerializer < ActiveModel::Serializer
```

Comment:
I moved these files to use our serializers everywhere we render JSON. I have this as a separate, final commit so it can be dropped if it's not something we want. I wasn't sure if it was an intentional decision to only serialize with these attributes in our Index actions. I described this in commit d93ac04